<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserHome</class>
 <widget class="QMainWindow" name="UserHome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton {
	
	background-color: rgb(255, 255, 255);
}
</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>210</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>111</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/img/SmallCardVaultLogo.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="setsCombo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Pokemon</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Yu-Gi-Oh!</string>
      </property>
     </item>
    </widget>
    <widget class="QListWidget" name="setsList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>171</width>
       <height>431</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>1070</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #d9c57e;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>40</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="placeholderText">
      <string>Sort/Filter</string>
     </property>
     <item>
      <property name="text">
       <string>Your Gay</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Aryas Gay</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>40</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Add New Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>40</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Add New Card</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="cardsFrame">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>1071</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 250, 250);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>931</width>
       <height>491</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>929</width>
        <height>489</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
